packages/opencensus-instrumentation-grpc/test/test-grpc.ts

```diff
@@ -546,7 +546,7 @@ describe('GrpcPlugin() ', function() {
                 grpcModule.status.OK
               );
               if (method.method === grpcClient.unaryMethod) {
-                assertStats(testExporter, 170);
+                assertStats(testExporter, 188);
```

Comment:
Out of curiosity, why are the numbers changing here? Will that confuse users at all?

Reply:
The latest gRPC release (`grpc@1.24.2`) added new options (`flags`) to Metadata class due to this size of bytes send increased a little bit (we measure the sent_bytes_per_rpc).

Proposal: https://github.com/grpc/proposal/blob/master/L48-node-metadata-options.md
PR: https://github.com/grpc/grpc-node/pull/796